Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Details.cshtml

```diff
@@ -121,29 +121,26 @@
             </div>
         }
 
+
         @if (Model.IsClosed)
         {
-            if (Model.Establishment.CloseDate != null)
-            {
-                <div id="gias-establishment-closed-message" class="gias-establishment-closed-message">
-                    <p>
-                        This establishment closed on @(Model.Establishment.CloseDate?.ToString("d MMMM yyyy")).
-                        View details of <a href="#school-links" class="tab-manipulator">linked establishments</a>
-                    </p>
-                </div>
-            }
-            else
-            {
-                <div id="gias-establishment-closed-message" class="gias-establishment-closed-message">
-                    <p>
-                        This establishment is closed.
-                        View details of <a href="#school-links" class="tab-manipulator">linked establishments</a>
-                    </p>
-                </div>
-            }
+            <div id="gias-establishment-closed-message" class="gias-establishment-closed-message">
+                <p>
+                    @if (Model.TabDisplayPolicy.Links)
+                    {
+                        <span>
+                            @Model.ClosedStatusMessage
+                            View details of<a href="#school-links" class="tab-manipulator"> linked establishments</a>
+                        </span>
+                    }
+                    else
+                    {
+                        @Model.ClosedStatusMessage
+                    }
+                </p>
+            </div>
         }
-    </div>
-</div>
```

Comment:
These two closing `</div>`  bits need to return, to close the opening tags on lines 97/98

```cshtml
<div class="govuk-grid-row">
    <div class="govuk-grid-column-full">
```